i18n/proposals.i18n.dart

```diff
@@ -4,35 +4,36 @@ extension Localization on String {
 
 static var _t = Translations.byLocale("en_us") +
     {
-      "es_es": {
+      "pt_br": {
         
-        "Proposals": "Propuestas",
-        'Staged': "Planeadas",
-        'Evaluate': "Evaluar",
-        "Open": "Abiertas",
-        'Passed': "Aprobadas",
-        'Failed': "Fallidas",
+        "Proposals": "Propostas",
+        'Staged': "Agendadas",
+        'Evaluate': "Avaliar",
+        "Open": "Abertas",
+        'Passed': "Aprovadas",
+        'Failed': "Rejeitadas",
 
-        'Created by:': "Creada por",
+        'Created by:': "Criado por",
         'total\n%s': "votos\n%s",
         'Voted': "Votado",
-        'Yes': "Sí",
-        'No': "No",
+        'Yes': "Sim",
+        'No': "Não",
 
-        'Recipient: %s ': "Recipiente: %s ",
-        'Requested: %s SEEDS': "Monto solicitado: %s SEEDS",
+        'Recipient: %s ': "Destinatário: %s ",
+        'Requested: %s SEEDS': "Solicitado: %s SEEDS",
+        'Funded by: %s ': "Financiado por: %s",
         'Type: %s ': "Tipo %s ",
-        'Alliance': "Alliance",
-        'Campaign': "Campaign",
+        'Alliance': "Alliança",
```

Comment:
It's special word in SEEDs? in case not, correct be "Aliança" with one 'l' letter. @juliolrmonteiro

Reply:
Fixed on latest commit, thanks!